packages/babel-plugin-transform-es2015-classes/test/fixtures/loose/accessing-super-properties/expected.js

```diff
@@ -6,8 +6,8 @@ var Test = function (_Foo) {
 
     var _this = babelHelpers.possibleConstructorReturn(this, _Foo.call(this));
 
-    _Foo.prototype.test;
-    _Foo.prototype.test.whatever;
+    babelHelpers.get(Object.getPrototypeOf(Test.prototype), "test", _this);
```

Comment:
Since we are in loose mode, here it would be better to use `babelHelpers.get(_Foo, "test", _this)`.

Reply:
That would be accessing the static `test` property wouldn't it? The reason for using `Test.prototype` is to actually simulate `super` property access, not just normal property access.

Reply:
Whoops, you are right. I meant `babelHelpers.get(_Foo.prototype, "test", _this)`.
Since we are in loose mode, we can assume that the user didn't modify Test's prototype, so this is true:
```js
class Foo {}
class Test extends Foo {}

console.log(Object.getPrototypeOf(Test.prototype) === Foo.prototype);
```